database/postgresconn/postgresconn_test.go

```diff
@@ -504,8 +425,8 @@ func TestCheckBeforeCreateTable(t *testing.T) {
 
 		// revoke privileges
 		mustRun(t, d, []string{
-			"REVOKE CREATE ON SCHEMA barfoo FROM PUBLIC",
-			"REVOKE CREATE ON SCHEMA barfoo FROM not_owner",
+			"REVOKE CREATE ON SCHEMA barfoo FROM PUBLIC;",
```

Comment:
just curious if the new line termination requirement can cause existing inputs to fail? Is it possible to test that?

Reply:
No, existing inputs were only in this library, all of our migrations were using the parser that required the line ending markers, this is a result of removing an extra code path so overall this is an improvement as all tests will be using the same code path and not different ones.